pkg/boot/linux.go

```diff
@@ -57,7 +57,7 @@ func (li *LinuxImage) String() string {
 }
 
 func copyToFile(r io.Reader) (*os.File, error) {
-	f, err := ioutil.TempFile("", "nerf-netboot")
+	f, err := ioutil.TempFile("", "linux-image")
```

Comment:
If we are kexec'ing a plan 9 kernel, doesn't the name nerf-netboot make more sense?

Reply:
I was only wondering if netboot is the right term. I integrated the OSImage interface in stboot some time ago (where there is a kind of local boot option as well using boot.LinuxImage) and it always felt strange to read "..netboot" in the Logs. Same must be true for grub parsers etc., I guess.
"linux-image" is just a suggestion. 

Reply:
very good point. hmm. kexec-image?

Reply:
kexec-image would be good. I copied this code at some point from a netboot-only command, sorry for the name :)